src/daemon/command_parser_executor.cpp

```diff
@@ -186,12 +188,35 @@ bool t_command_parser_executor::print_block(const std::vector<std::string> &args
 {
 	if(args.empty())
 	{
-		std::cout << "expected: print_block (<block_hash> | <block_height>)" << std::endl;
+		GULPS_PRINT("expected: print_block (<block_hash> | <block_height>)");
 		return false;
 	}
 
 	const std::string &arg = args.front();
-	try
+
+	//TEMPORARY (?)
```

Comment:
@fireice-uk  please check this code snipped again since you marked it with the question `TEMPORARY?`

Reply:
Code looks ok, removed the comment.